scripts/src/test.sh

```diff
@@ -58,18 +58,20 @@ gtest() {
 }
 
 # $1 - ROOT DIR
-# $2 - yamls list (optional)
+# $2 - TARGET DIR
+# $3 - yamls list (optional)
 rtest() {
   ROOT_DIR="$1"
+  TARGET_DIR="$2"
 
-  YAMLS=$(parse_yamls_list "$2")
+  YAMLS=$(parse_yamls_list "$3")
```

Comment:
That's already solved in more correct way, merge master, please

Reply:
It is already merged. `TARGET_DIR` is required there to put resulting `junit.xml` so arguments count increased